docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,76 @@
+# Introducing the new Azure SDK Management Libraries for JavaScript
+
+We're excited to announce the new GA release for JavaScript libraries that enable you to provision and manage Azure resources via the Azure Resource Manager. These libraries have several new features that help developers to have a great experience in using the Azure Services. Some of the important libraries that have been released are:
+
+1. [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute)
+2. [@azure/arm-network](https://www.npmjs.com/package/@azure/arm-network)
+3. [@azure/arm-storage](https://www.npmjs.com/package/@azure/arm-storage)
+4. [@azure/arm-sql](https://www.npmjs.com/package/@azure/arm-sql)
+5. [@azure/arm-resources](https://www.npmjs.com/package/@azure/arm-resources)
+6. [@azure/arm-policy](https://www.npmjs.com/package/@azure/arm-policy)
+7. [@azure/arm-managedapplications](https://www.npmjs.com/package/@azure/arm-managedapplications)
+8. [@azure/arm-links](https://www.npmjs.com/package/@azure/arm-links)
+9. [@azure/arm-locks](https://www.npmjs.com/package/@azure/arm-locks)
+10. [@azure/arm-features](https://www.npmjs.com/package/@azure/arm-features)
+
+These libraries follow the new [Azure Library guidelines](https://azure.github.io/azure-sdk/general_introduction.html) that were written to provide an improved and consistent development experience across Azure services. The [TypeScript & JavaScript specific guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) that are also followed ensure that these libraries have a natural and idiomatic feel to the JavaScript and TypeScript ecosystem. These libraries have been generated using the new [Autorest Typescript Library Generator](https://www.npmjs.com/package/@autorest/typescript). They provide new authentication mechanisms, better paging and long running operations experience.. We encourage you to try out the libraries and provide your feedback.
+
+## Key concepts
+
+While a detailed list and usage of the new features can be found in the migration guide and the quick start guide can be used for first time users, below is an overview of the major changes in the new libraries
```

Comment:
Add links to migration guide and quick start guide here

Reply:
Done